nextflow.config

```
@@ -1,212 +1,224 @@
+    Default config options for all compute environments
+----------------------------------------------------------------------------------------
+*/
 
 // Global default params, used in configs
 params {
 
-  // Workflow flags
-  input = "data/*.fastq.gz"
-  input_paths = null
-  outdir = './results'
-  protocol = 'illumina'
-
-  // Reference genomes
-  genome = null
-  mirna_gtf = null
-  bt_indices = null
-  mirtrace_protocol = false
-  references_parsed = false
-  mature = "ftp://mirbase.org/pub/mirbase/CURRENT/mature.fa.gz"
-  hairpin = "ftp://mirbase.org/pub/mirbase/CURRENT/hairpin.fa.gz"
-
-  // Trimming options
-  clip_r1 = 0
-  three_prime_clip_r1 = 0
-  three_prime_adapter = "TGGAATTCTCGGGTGCCAAGG"
-  min_length = 17
-  skip_qc = false
-  skip_fastqc = false
-  skip_multiqc = false
-  skip_mirdeep = false
-  save_reference = true
-  seq_center = ""
-  trim_galore_max_length = 40
-
-  // Boilerplate options
-  multiqc_config = false
-  email = false
-  email_on_fail = false
-  max_multiqc_email_size = 25.MB
-  plaintext_email = false
-  monochrome_logs = false
-  help = false
-  igenomes_base = 's3://ngi-igenomes/igenomes'
-  tracedir = "${params.outdir}/pipeline_info"
-  igenomes_ignore = false
-  custom_config_version = 'master'
-  custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
-  hostnames = false
-  config_profile_name = null
-  config_profile_description = false
-  config_profile_contact = false
-  config_profile_url = false
-  publish_dir_mode = 'copy'
-  validate_params = true
-  show_hidden_params = false
-  schema_ignore_params = 'genomes,input_paths'
-
-  // Defaults only, expecting to be overwritten
-  max_memory = 128.GB
-  max_cpus = 16
-  max_time = 240.h
+    // Input options
+    input                      = null
+
+    // Workflow flags
+    outdir                     = './results'
+    protocol                   = 'illumina'
+
+    // References
+    genome                     = null
+    igenomes_base              = 's3://ngi-igenomes/igenomes'
+    igenomes_ignore            = false
+    fasta                      = null
+    mirna_gtf                  = null
+    bt_indices                 = null
+    mirtrace_species           = null
+    mirtrace_protocol          = 'illumina'
+    mature                     = "ftp://mirbase.org/pub/mirbase/CURRENT/mature.fa.gz"
+    hairpin                    = "ftp://mirbase.org/pub/mirbase/CURRENT/hairpin.fa.gz"
+
+    // Trimming options
+    clip_r1                    = 0
+    three_prime_clip_r1        = 0
+    three_prime_adapter        = "TGGAATTCTCGGGTGCCAAGG"
+    min_length                 = 17
+    skip_qc                    = false
+    skip_fastqc                = false
+    skip_multiqc               = false
+    skip_mirdeep               = false
+    skip_trimming              = false
+    save_reference             = true
```

Shouldn't be this by default set to false instead?
```suggestion
    save_reference             = false
```